workflows/sars-cov-2-variant-calling/sars-cov-2-ont-artic-variant-calling/ont-artic-variation.ga

```diff
@@ -750,59 +750,77 @@
         },
         "15": {
             "annotation": "",
-            "content_id": "toolshed.g2.bx.psu.edu/repos/iuc/lofreq_filter/lofreq_filter/2.1.5+galaxy0",
+            "content_id": "toolshed.g2.bx.psu.edu/repos/iuc/snpeff_sars_cov_2/snpeff_sars_cov_2/4.5covid19",
             "errors": null,
             "id": 15,
             "input_connections": {
-                "invcf": {
+                "input": {
                     "id": 13,
                     "output_name": "out_annotated"
                 }
             },
-            "inputs": [],
-            "label": null,
-            "name": "Lofreq filter",
+            "inputs": [
+                {
+                    "description": "runtime parameter for tool SnpEff eff:",
+                    "name": "intervals"
+                },
+                {
+                    "description": "runtime parameter for tool SnpEff eff:",
+                    "name": "transcripts"
+                }
+            ],
```

Comment:
Is this correct @wm75 ? That looks like a bug we've fixed in https://github.com/galaxyproject/galaxy/pull/11652

Reply:
This one looks correct. SnpEff really has these two optional runtime parameters. These are rather annoying because they prevent WFs with SnpEff from being runnable from the simplified WF run dialog, but the block existed also in the previous version (see the complementary diff further down), it's just that it shows up here as a change because of the altered tool order.